library/eclipse-mosquitto

```diff
@@ -1,7 +1,11 @@
 Maintainers: David Audet <david.audet@ca.com> (@daudetCA)
 GitRepo: https://github.com/eclipse/mosquitto.git
 
-Tags: 1.4.10, latest
+Tags: 1.4.12, latest
+GitCommit: 0bb602ed7a89da80d25e6b959a130fdcf0556be5
+Directory: docker/1.4.12
+
+Tags: 1.4.10
```

Comment:
Any particular reason to keep this older version around?  It won't be updated, right?  (We won't delete the already-pushed tags, but the list included here controls what we rebuild and what gets listed under "Supported tags" on the Hub image description)